packages/core/http/core-http-server-internal/src/types.ts

```diff
@@ -61,8 +61,6 @@ export interface InternalHttpServiceSetup
     contextName: ContextName,
     provider: IContextProvider<Context, ContextName>
   ) => IContextContainer;
-
-  registerPrebootRoutes(path: string, callback: (router: IRouter) => void): void;
```

Comment:
This API was only used to register the old status handler to the preboot server during setup (yeah...). Given I removed the last usage of it (now the handler is registered from the service's `preboot` stage)